Dockerfile.openshift

```diff
@@ -1,9 +1,9 @@
-FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.22-openshift-4.17 AS builder
+FROM registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.23-openshift-4.19 AS builder
 WORKDIR /go/src/github.com/openshift/secrets-store-csi-driver
 COPY . .
 RUN make build
 
-FROM registry.ci.openshift.org/ocp/4.17:base-rhel9
+FROM registry.ci.openshift.org/ocp/4.19:base-rhel9
```

Comment:
@chiragkyal can you update `Dockerfile.bats` and `Dockerfile.e2eprovider` to use the same 4.19 images as `Dockerfile.openshift`?

The ART bot will automatically bump images that it builds (i.e. `Dockerfile.openshift`) but the other two dockerfiles do not get updated automatically because they are only used in our presubmit jobs for testing.

Reply:
Thanks for the suggestion. I've updated the `Dockerfile.bats` and `Dockerfile.e2eprovider` to 4.19